src/Module/module.css

```diff
@@ -1,84 +1,25 @@
 @import "../proto/base.css";
 @import "../proto/definitions/_colors.css";
 
-$moduleSpacing:  3rem;
-
-
-%module-highlight
-{
-    display:            block;
-    content:            '';
-    position:           absolute;
-
-    left:               0;
-    top:                0;
-
-    height:             100% ;
-    width:              100%;
-
-    background:         var(  --GS-9  );
-
-    opacity:            0;
-    transition:         opacity 200ms ease-in-out;
-}
-
 .default
 {
     /* this needs to be a flex column for vertical alignment
     * of header w/min-height (flexbugs #3) */
     display:        flex;
     flex-direction: column;
 
-    margin-bottom:  $moduleSpacing;
-
     background:     var( --PC-WHITE );
 
     box-shadow:     0 0.1rem 0.4rem 0 var( --GS-8 );
-
-    .default /* nested module */
-    {
-
-        margin-top: 2rem;
-
-        /* negate the module spacing when collapsed submodules are stacked */
-        &.default + .default
-        {
-            margin-top: -$moduleSpacing;
-        }
-    }
-}
-
-.level__2
-{
-    & > .header + .content
-    {
-        padding-top:    1rem;
-    }
-}
-
-.level__3,
-.level__4
-{
-    & > .header,
-    & > .content
-    {
-        padding-top:    2rem;
-        padding-bottom: 2rem;
-    }
 }
 
 .collapsible
 {
     & > .header
     {
-        cursor: pointer;
-
-        &::before
-        {
-            @extend %module-highlight;
-            transition: height 200ms ease-in-out, opacity 200ms ease-in-out;
-            opacity:    1;
-        }
+        cursor:     pointer;
+        background: var(  --GS-11  );
+        transition: height 200ms ease-in-out, background-color 200ms ease-in-out;
```

Comment:
The transition does not need to be defined here. It’s already defined for the `.header` on line 62.

(Also `height` is no longer needed... that must be left over from some earlier version of the Module.)